source/core/NumericString.cpp

```diff
@@ -418,7 +422,7 @@ static Int32 GetTimeZoneOffsetFromTimeAndLocale(StaticTypeAndData *arg, const Fo
 /**
  * main format function, all the %format functionality is done through this one
  * */
-void Format(SubString *format, Int32 count, StaticTypeAndData arguments[], StringRef buffer, ErrorCluster *errPtr)
+void Format(SubString *format, Int32 count, StaticTypeAndData arguments[], StringRef buffer, ErrorCluster *errPtr, SubString* formatName)
```

Comment:
You may want to make the new parameter being added as a default (optional) parameter with default value of nullptr. This will help avoid fixing all the existing callers.

Reply:
Done